prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/ConstantResolver.kt

```diff
@@ -339,9 +340,11 @@ object CustomFHIRFunctions {
     private val oidRegex = """^[0-2](.(0|[1-9]\d*))+$""".toRegex()
 
     /**
-     * Regex to identify CLIAs.  CLIAs are just 10 alphanumeric characters.
+     * Regex to identify CLIAs.
+     * CLIAs are defined as "10-digit alphanumeric numbers, with the "D"in the third position.
+     * One exception is DoD numbers, where the first character is also "D".
      */
-    private val cliaRegex = """^[a-zA-Z\d]{10}$""".toRegex()
+    private val cliaRegex = """^[dD\d]\dD\d{7}$""".toRegex()
```

Comment:
Let's change something here.  From an earlier discussion in Slack about it, maybe lets do any uppercase letter or a digit for those two positions.  The reason is that they said a valid clia can be like Z1D111111 for DOD.  Might as well allow any letter since no one has a spec for this.